.github/workflows/update_bee.yaml

```
@@ -27,6 +27,7 @@ jobs:
 
           echo "IMAGE_VERSION=$image_version" >> $GITHUB_ENV
           echo "BEE_VERSION=${image_version/-stateful}" >> $GITHUB_ENV
+          echo "TARGET_BRANCH=${BEE_VERSION/-rc*}" >> $GITHUB_ENV
```

shouldn't it have the `chore/` prefix?
```suggestion
          echo "TARGET_BRANCH=chore/${BEE_VERSION/-rc*}" >> $GITHUB_ENV
```

---

Well I didn't want to add it. The idea is that this branch will be the base branch for all that Bee's version work to which everything that needs this specific Bee's version will be merged. Once final Bee version is released than this will get merged to master...